.github/workflows/deploy-fastly-snippets.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     env:
       FASTLY_SERVICE_ID: ${{ secrets.FASTLY_SERVICE_ID }}
       FASTLY_API_TOKEN: ${{ secrets.FASTLY_API_TOKEN }}
-      DELETE_SNIPPETS: ${{ github.event.inputs.delete-fastly-snippets }}
+      DELETE_SNIPPETS: ${{ github.event.inputs.delete-snippets }}
```

Comment:
This should match [this line](https://github.com/guardian/support-frontend/pull/5472/files#diff-bea53c7289b23da8fb6a2423001931ea447fcc492bd92bfb559844177ba8d40dR9).